swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,id=None, condition=0, age=0):
```

Comment:
```suggestion
    def __init__(self, id=None, condition=0, age=0):
```
Nit: You're missing a whitespace after the first comma. Be mindful to use/exclude whitespaces where necessary. It is easy to overlook, but having inconsistent use of whitespaces leads to a messy codebase.

Here's the section on [whitespaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) in the PEP8 styleguide.